app/javascript/components/server-components/FollowersPage.tsx

```diff
@@ -58,6 +59,86 @@ const Layout = ({
   );
 };
 
+const ExportFollowers = ({ close }: { close: () => void }) => {
+  const [followers, setFollowers] = React.useState(true);
+  const [customers, setCustomers] = React.useState(false);
+  const [affiliates, setAffiliates] = React.useState(false);
+
+  const handleDownload = async () => {
+    try {
+      const response = await request({
+        url: Routes.audience_export_path(),
```

Comment:
Looks like we have another place that currently calls the export path

https://github.com/antiwork/gumroad/blob/919fb3763f6317ba9599d928a981542e4f2b82c5/app/javascript/components/server-components/FollowersPage.tsx#L144

Let's make sure that other place also:

- Pass in the correct params
- Reflect the new "You’ll receive an email with the download link shortly." behavior.